proto/aep-api/aep/api/resource.proto

```diff
@@ -0,0 +1,54 @@
+// Copyright 2024 aep-dev. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package aep.api;
+
+import "google/protobuf/descriptor.proto";
+
+option go_package = "github.com/aep-dev/aep-go/api;api";
+
+extend google.protobuf.MessageOptions {
+  // A descriptor for a resource type.
+  ResourceDescriptor resource = 1059;
```

Comment:
this should use a field extension in the range 1264-1274 (whatever is currently unused in this repository). This is part of the extension reserved by the aeps: https://github.com/protocolbuffers/protobuf/blob/main/docs/options.md